packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -280,7 +281,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         if (!this.channelDeferred) {
             this.channelDeferred = new Deferred<IFluidDataStoreChannel>();
             this.realizeCore().catch((error) => {
-                this.channelDeferred?.reject(error);
+                this.channelDeferred?.reject(CreateProcessingError(error, undefined));
```

Comment:
Can you please add comment, i.e.
, undefined /* message */);

Reply:
yes. something i didn't do here, but am considering is storing the details (seq, refSeq, timestamp,...) of the last message a channel receives in it's attributes such that we can provide it here, a give a context to the error. Timestamp is especially important for alerting, as it allows us to differentiated old broken documents for new incidents breaking documents. this would however impact blob deduping. curious your thoughts on that?

Reply:
For now maybe just make that param optional...?